Gems/AzQtComponentsForPython/bindings_azqtcomponents.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#pragma once
+/*
+#include <QPointer>
+
+//#include <AzCore/RTTI/TypeInfo.h>
+//#include <AzCore/Math/MathUtils.h>
+#include <AzCore/Math/Color.h>
+#include <AzCore/std/function/function_template.h>
+#include <AzCore/std/chrono/chrono.h>
+*/
+
+//#include <AzQtComponents/Components/AutoCustomWindowDecorations.h>
+
+// Ensure the correct AZ_VE_NUM_ARGS macro is used inside variadic.h
+#define FORCE_VA_OPT_ARG 1
+///////////////////////
+#include <AzQtComponents/Components/ButtonDivider.h>
+#include <AzQtComponents/Components/ButtonStripe.h>
+#include <AzQtComponents/Components/ConfigHelpers.h>
+#include <AzQtComponents/Components/DockBar.h>
+#include <AzQtComponents/Components/DockBarButton.h>
+#include <AzQtComponents/Components/DockMainWindow.h>
+#include <AzQtComponents/Components/DockTabBar.h>
+#include <AzQtComponents/Components/DockTabWidget.h>
+
+#include <AzQtComponents/Components/ExtendedLabel.h>    
+#include <AzQtComponents/Components/FancyDocking.h>
+#include <AzQtComponents/Components/FancyDockingDropZoneWidget.h>
+#include <AzQtComponents/Components/FancyDockingGhostWidget.h>
+#include <AzQtComponents/Components/FlowLayout.h>
```

Comment:
There are a few commented out lines here, do they need to be removed? If the files exist they won't show up in the project

Reply:
Removed.